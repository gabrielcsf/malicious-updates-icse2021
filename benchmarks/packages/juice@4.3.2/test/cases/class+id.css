#test {
  border: 1px solid #f00;
}
.woot {
  display: inline-block;
}
