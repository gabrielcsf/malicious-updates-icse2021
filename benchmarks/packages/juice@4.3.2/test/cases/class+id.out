<html><body>
<p id="test" class="woot" style="display: inline-block; border: 1px solid #f00;">Woot</p>
</body></html>
